### PR TITLE
MOBILE-179

### PR DESCRIPTION
**Pull request by martyu**
Pull out "play queue" functionality from VLCAudioPlayer

Add "offline dev mode" with stubs (the plane didn't have internet so i threw this in on the flight to unblock.)

### CODE REVIEW

File: iosApp/Modules/Screens/Library/UI/FilterView.swift [hunk 1]
```diff
@@ -0,0 +1,38 @@
+//import SwiftUI
+//import shared
+//import AudioPlayer
+//
+//struct FilterSortMenuView: View {
```
Comment (cristhianescobar) [inline iosApp/Modules/Screens/Library/UI/FilterView.swift]: Can we remove the code we are using here?

File: iosApp/Modules/Screens/NowPlaying/UI/NowPlayingView.swift [hunk 2]
```diff
@@ -24,6 +24,14 @@ public struct NowPlayingView: View {
 		}
 		.background(background)
 		.backButton()
+		//TODO: make an "ErrorProviding" protocol, replace all these.
```
Comment (cristhianescobar) [inline iosApp/Modules/Screens/NowPlaying/UI/NowPlayingView.swift:27]: same here, can this be remved?

File: iosApp/Modules/Mocks/MockData/Music/getSchwifty1.mp3
Comment (cristhianescobar) [inline iosApp/Modules/Mocks/MockData/Music/getSchwifty1.mp3:1]: Can we remove these tracks, or at least ensure they are not being deployed with the app when we send out a build. We can only play tracks owned by the user's wallet.

**Review by cristhianescobar — CHANGES_REQUESTED** [review]
Can we ensure the hard-coded tracks are not available to user? We should only play tracks they owned. If this is just for testing that is fine, we just cannot ship this tracks out as we don't as we cannot pay royalties from the app yet.

File: iosApp/Modules/Screens/Library/UI/FilterView.swift (hunk 1 above)
Comment (martyu) [reply iosApp/Modules/Screens/Library/UI/FilterView.swift]: bah, yep, that was the chatgpt experiment.  it did ok 😐

File: iosApp/Modules/Screens/NowPlaying/UI/NowPlayingView.swift (hunk 2 above)
Comment (martyu) [reply iosApp/Modules/Screens/NowPlaying/UI/NowPlayingView.swift:27]: I still need to add this back in but there were issues with the audio player errors I need to fix first.

File: iosApp/Modules/Mocks/MockData/Music/getSchwifty1.mp3
Comment (martyu) [reply iosApp/Modules/Mocks/MockData/Music/getSchwifty1.mp3:1]: They're only in debug builds

Comment (martyu) [reply iosApp/Modules/Mocks/MockData/Music/getSchwifty1.mp3:1]: Only there so we can continue development when there's no internet (I use it on the plane sometimes)

**Pull request merged**